### DISCUSSION TITLE
CoPilot user management in larger companies (CoPilot for Business)

### DISCUSSION THREAD

**Discussion by andbmg** [community/community#64500] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi there

I work in a larger company and we are currently testing CoPilot for Business. It's a bit tricky to rollout CoPilot for all developers in our company (~100) since the user management includes a lot of manual steps. 

I've seen that there is an upload for several users at the same time. But how do you guys manage your company's user? E.g. when somebody leaves the company. Do you manage this manually or are you waiting for the API?

**Comment by arafajs** [comment, marked as answer]
- Use the bulk user upload feature to add multiple users to Copilot at once. This can be found in the "Manage users" section of the Copilot for Business settings page.
- Create a script to automate the user management process. This can be a helpful way to avoid making mistakes and to ensure that all of your users are properly provisioned.
- Use the Copilot for Business API to manage user access programmatically. This is the most flexible option, but it also requires some technical knowledge.

**Reply by andbmg** [reply]
Thank you for your fast reply, @arafajs!
